code/model/DNEnvironment.php

```diff
@@ -615,9 +615,18 @@ public function CurrentBuild() {
 
 	/**
 	 * A list of past deployments.
-	 * @return ArrayList
+	 * @param string $orderBy - the name of a DB column to sort in descending order
+	 * @return \ArrayList
 	 */
-	public function DeployHistory() {
+	public function DeployHistory($orderBy = '') {
+
+		$sort = [];
+		if($orderBy != '') {
```

Comment:
Can we stick with the `if ($expression) {` syntax? The whitespace after the `if`, `foreach` or whatever?